### ISSUE TITLE
Profiling result

### ISSUE THREAD

**Issue by zoenagy235** [dreamnettech/dreampower#95]
Conversion of one image takes 56secs on i5 x64 (CPU only).
62% of this time is spent in torch.nn.Conv2d (Applies a 2D convolution over an input signal composed of several input planes.)

Is there anyway to optimize this?

Some ideas I found:

> Fast conv2d (random weights) Time elapsed:  0.08310413360595703
> Slow conv2d (trained weights) Time elapsed:  3.3756301403045654
> 
> maybe you're hitting a problem with denormalized weights? you may try setting torch.set_flush_denormal(True) and see if that helps
> this is indeed because some of the weights are in denormal range, and Intel / ARM CPUs take a very slow path for floating point math in such cases. 

It didnt work for me, are the weights normalised for performance?

[profiling.txt](https://github.com/dreamnettech/dreampower/files/4469649/profiling.txt)
